lib/src/progress_bar.dart

```diff
@@ -57,75 +57,87 @@ class _VideoProgressBarState extends State<VideoProgressBar> {
     super.deactivate();
   }
 
-  void _seekToRelativePosition(Offset globalPosition) {
+  Duration _calcRelativePosition(Offset globalPosition) {
     final box = context.findRenderObject()! as RenderBox;
     final Offset tapPos = box.globalToLocal(globalPosition);
     final double relative = tapPos.dx / box.size.width;
     final Duration position = controller.value.duration * relative;
-    controller.seekTo(position);
+
+    return position;
+  }
```

Comment:
This should actually be a extension on `BuildContext`. In other words, this should take in both a `Duration` and `Offset` and compute the resulting position Duration.